itou/metabase/management/commands/populate_metabase_emplois.py

```diff
@@ -470,11 +471,17 @@ def populate_enums(self):
         # TODO(vperron,dejafait): This works as long as we don't have several table creations in the same call.
         # If we someday want to create several tables, we will probably need to disable autocommit in our helper
         # functions and make it manually at the end.
-        table_name = "c1_ref_origine_candidature"
-        self.stdout.write(f"Preparing content for {table_name} table...")
-        rows = [OrderedDict(code=str(item), label=item.label) for item in Origin]
-        df = get_df_from_rows(rows)
-        store_df(df=df, table_name=table_name)
+        enum_data = [
+            {"table_name": "c1_ref_origine_candidature", "enum": Origin},
+            {"table_name": "c1_ref_type_contrat", "enum": ContractType},
+        ]
+        for item in enum_data:
+            table_name = item["table_name"]
+            enum = item["enum"]
```

Comment:
Avec un peu moins de code :
```suggestion
        enum_to_table = {
            Origin: "c1_ref_origine_candidature",
            ContractType: "c1_ref_type_contrat",
        }
        for enum, table_name in enum_to_table.items():
```
La logique peux être inversé si ça vous semble plus naturel :).